openlibrary/core/imports.py

```diff
@@ -63,7 +102,8 @@ def find_by_identifier(identifier):
             return ImportItem(result[0])
 
     def set_status(self, status, error=None, ol_key=None):
-        logger.info("set-status %s - %s %s %s", self.ia_id, status, error, ol_key)
+        _id = self.ia_id or "%s:%s" % (self.batch_id, self.id)
```

Comment:
This should be a trailing underscore, leading means private var, trailing is for avoiding name conflict by convention. https://hackernoon.com/understanding-the-underscore-of-python-309d1a029edc

Reply:
```suggestion
        id_ = self.ia_id or "%s:%s" % (self.batch_id, self.id)
```